globus_sdk/exc.py

```diff
@@ -133,6 +134,30 @@ def _load_from_json(self, data):
         self.request_id = data["request_id"]
 
 
+class AuthAPIError(GlobusAPIError):
+    """
+    Error class for the API components of Globus Auth.
+
+    Customizes JSON parsing.
+    """
+    def _load_from_json(self, data):
+        """
+        Load error data from a JSON document.
+
+        Sets `code` statically because Auth doesn't have API error codes, and
+        looks for a top-level "error" attribute.
+        """
+        self.code = "Error"
```

Comment:
Looks like the tests are failing on this static setting of self.code.

I would expect something like the following to pass tests, but if the tests themselves should be changed I can do that.

`if "code" in data:`
    `self.code = data["code"]`
`else:`
    `self.code = "Error"`